lib/ansible/module_utils/yumdnf.py

```diff
@@ -92,6 +98,20 @@ def __init__(self, module):
         self.enablerepo = self.listify_comma_sep_strings_in_list(self.enablerepo)
         self.exclude = self.listify_comma_sep_strings_in_list(self.exclude)
 
+        # This should really be redefined by both the yum and dnf module but a
+        # default isn't a bad idea
+        self.lockfile = '/var/run/yum.pid'
+
+    def wait_for_lock(self):
+        '''Poll until the lock is removed if interval is a positive number'''
+        if os.path.isfile(self.lockfile) and self.lock_timeout > 0:
```

Comment:
Won't this check fail for `dnf` since it is using a glob?

Reply:
@sivel it sure will, I'll fix.